hw added

MozesHunor opened the pull request:
Alex Boga didn't accept my collaborator request so I couldn't add him.

kxkdjhd committed:
hw added

andrei-hegedus commented on dealership/src/main/java/ro/scit/dealership/Car.java:
```diff
@@ -0,0 +1,11 @@
+package main.java.ro.scit.dealership;
+
+public class Car {
+    String model;
```
Encapsulation with access modifiers?

andrei-hegedus commented on dealership/src/main/java/ro/scit/dealership/Green.java:
```diff
@@ -0,0 +1,8 @@
+package main.java.ro.scit.dealership;
+
+public class Green {
```
GreenBudgetProrgam would be a better name.

andrei-hegedus reviewed (changes requested):
See my comments.
Also: int this pull-request there are 41 files out of which just 5 needs to be reviewed. Please avoid this.

Use .gitignore.

kxkdjhd committed:
Applied changes: 1. used .gitignore
                                 2. used encapsulation with access modifiers
                                 3. renamed class Green to GreenBudgetProgram
